clients/csharp-dotnet-core/voice-assistant-test/README.md

```diff
@@ -306,6 +309,8 @@ In summary, to accurately measure UPL do these three things:
 
 Note: This has not been tested when keyword activation is used. At this time we do not recommend measuring UPL using the above method when [Keyword](#Keyword) is set to true.
 
+### Running tests twice once with text input and once with audio
+The [WavAndUtterancePairs](#WavAndUtterancePairs) flag, if set to true will run the same test twice once with audio and again with text. A turn with audio input may fail with due to incrorect speech recognition result but may pass with text input. 
 

```

Comment:
"with due"? remove "with".
I see that you are a man of few words. The topics section is where you are supposed to explain more, and perhaps give an example. but that’s okay, I may update it later.

Reply:
Done